### PR TITLE
This PR resolves Issue 221

### PR DESCRIPTION
**Pull request by ghost**
This PR relates to #221

The pull request consists of:

* Changes in AbstractIntegrationTest, DnaCaptureVarFastqIntegrationTest and thymeleaf templates to run
DnaCaptureVarFastqIntegrationTest in master mode
*  New thymeleaf template master_template_test for validation master script

### CODE REVIEW

File: src/integrationTest/resources/templates/DnaCaptureVarFastq/master_template_test.txt [hunk 1]
```diff
@@ -0,0 +1,107 @@
+#!/bin/bash -x
```
Comment (kamyshova) [inline src/integrationTest/resources/templates/DnaCaptureVarFastq/master_template_test.txt]: Why don't we reuse the master template for the integration test from the previous RnaExpressionFastqIntegrationTest?

Comment (ghost) [reply src/integrationTest/resources/templates/DnaCaptureVarFastq/master_template_test.txt]: Fixed

File: src/integrationTest/java/com/epam/fonda/DnaCaptureVarFastqIntegrationTest.java [hunk 2]
```diff
@@ -169,7 +290,21 @@ void testDnaCaptureVarFastq(String gConfigPath, String sConfigPath, String outpu
                         format("%s/DnaCaptureVar_Fastq_alignment_for_smv1_1_analysis.sh", OUTPUT_SH_FILES_DIR),
                         format("%s/%s/dnaCaptureVar_Fastq_alignment_for_smv1_1_analysis_template",
                                 DNA_CAPTURE_VAR_FASTQ_DIR,
-                                BWA_VARDICT_MUTECT1_STRELKA2_GATK_HAPOTYPECALLER_SCALPEL_LOFREQ_FREEBAYES_TEMPLATES)),
+                                BWA_VARDICT_MUTECT1_STRELKA2_GATK_HAPOTYPECALLER_SCALPEL_LOFREQ_FREEBAYES_TEMPLATES),
+                        new String[]{
```
Comment (kamyshova) [inline src/integrationTest/java/com/epam/fonda/DnaCaptureVarFastqIntegrationTest.java]: Add a space between `String[]` and `{` here and below, please

File: src/main/java/com/epam/fonda/utils/TestTemplateUtils.java [hunk 3]
```diff
@@ -0,0 +1,60 @@
+package com.epam.fonda.utils;
```
Comment (kamyshova) [inline src/main/java/com/epam/fonda/utils/TestTemplateUtils.java]: Add a license header, please

**Review by kamyshova — CHANGES_REQUESTED** [review]

File: src/main/java/com/epam/fonda/utils/TestTemplateUtils.java (hunk 3 above)
Comment (ghost) [reply src/main/java/com/epam/fonda/utils/TestTemplateUtils.java]: Done

File: src/integrationTest/java/com/epam/fonda/DnaCaptureVarFastqIntegrationTest.java (hunk 2 above)
Comment (ghost) [reply src/integrationTest/java/com/epam/fonda/DnaCaptureVarFastqIntegrationTest.java]: Done

**Review by kamyshova — APPROVED** [review]

**Pull request merged**